docs/3.4.0-release-notes.md

```diff
@@ -1,2 +1,3 @@
 ### Fixes
 + Support pushing to Azure Functions
++ Validate URL in Test Connection
```

Comment:
3.4.0 has already been released. You'll need to create a new 3.5.0 file

Reply:
3.5.0 release had been created.